### PR TITLE
Feature: deadline and phone number

### PR DESCRIPTION
**Pull request by FranzPoize**

### CODE REVIEW

File: src/components/ProjectDisplay/index.js [hunk 1]
```diff
@@ -368,13 +369,42 @@ class ProjectDisplay extends Component {
...
+										</TotalContainer>
+										<TotalContainer>
+											<ProjectTotal
+												sumDays={Math.ceil(
+													(new Date(
+														project.deadline,
+													).getTime()
+														- new Date().getTime())
+														/ 86400000,
+												)}
```
Comment (YoruNoHikage) [inline src/components/ProjectDisplay/index.js:387]: No need for `getTime`, operators use it automatically.

File: src/components/ProjectDisplay/index.js [hunk 2]
```diff
@@ -368,13 +369,42 @@ class ProjectDisplay extends Component {
...
+													<Plural
+														singular="jour"
+														plural="jours"
+														value={this.getProjectTotal(
+															project,
+														)}
+													/>
+												}
+											/>
+											{}
```
Comment (YoruNoHikage) [inline src/components/ProjectDisplay/index.js]: What is that?

File: src/containers/App/Project/create-project-form.js [hunk 3]
```diff
@@ -87,12 +110,65 @@ const SelectStyles = {
...
+const parseDate = (dateString) => {
+	const dates = dateString.split('/');
+
+	return new Date(`${dates[1]}/${dates[0]}/${dates[2]}`);
+};
+
 class CreateProjectForm extends React.Component {
+	constructor(props) {
+		super(props);
+	}
```
Comment (YoruNoHikage) [inline src/containers/App/Project/create-project-form.js]: No need

File: src/components/ProjectDisplay/index.js (hunk 2 above)
Comment (FranzPoize) [reply src/components/ProjectDisplay/index.js]: nobody knows

**Pull request merged**